eo-runtime/src/main/eo/org/eolang/positive-infinity.eo

```diff
@@ -181,17 +181,94 @@
   #  worked with multiple arguments without using objects from eo-collections.
   #  See positive-infinity.plus and negative-infinity.plus as an example
```

Comment:
@maxonfjvipon Do we need to remove this puzzle?

Reply:
@Graur yes, thanks